test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -201,7 +201,37 @@ Describe "ConsoleHost unit tests" -tags "Feature" {
             $observed[1] | Should Be "bar"
         }
 
-        It "-File should return exit code from script: <Filename>" -TestCases @(
+        It "-File should be able to pass bool values as strings to parameters: <BoolString>" -TestCases @(
```

Comment:
Yeah, that's a problem.  I've changed it so it only works with switches and added tests for other cases